course-structure.json

```diff
@@ -108,95 +90,93 @@
                             "lab": "Mint token rewards for reviews"
                         }
                     ]
-                },
+                }
+            ]
+        },
+        {
+            "title": "Solana tokens",
+            "slug": "solana-tokens",
+            "units": [
                 {
-                    "title": "Native Solana Program Development",
-                    "slug": "basic-solana-program-development",
+                    "title": "Introduction to Solana tokens",
+                    "slug": "introduction-to-solana-tokens",
                     "lessons": [
                         {
-                            "title": "Serialize program data",
-                            "slug": "serialize-instruction-data",
-                            "lab": "Build a React frontend to submit movie reviews"
-                        },
-                        {
-                            "title": "Deserialize program data",
-                            "slug": "deserialize-custom-data",
-                            "lab": "Build a React frontend to show movie reviews"
-                        },
-                        {
-                            "title": "Page, Order, and Filter program data",
-                            "slug": "paging-ordering-filtering-data",
-                            "lab": "Adding paging, ordering and searching to the movie reviews frontend"
-                        },
-                        {
-                            "title": "Hello World in native",
-                            "slug": "hello-world-program",
-                            "lab": "Create and deploy a native 'hello world' onchain program"
-                        },
-                        {
-                            "title": "Handle Instruction Data",
-                            "slug": "deserialize-instruction-data",
-                            "lab": "Make an onchain program that recieves movie reviews and deserialises them"
-                        },
-                        {
-                            "title": "State Management",
-                            "slug": "program-state-management",
-                            "lab": "Make an onchain program that stores movie reviews"
+                            "title": "Create tokens with the Token Program",
+                            "slug": "token-program",
+                            "lab": "Mint, transfer and burn tokens"
```

Comment:
We may want to update this chapter to use TEs once Explorer finally supports token extensions. However it sounds like it's a long way off - most recent update from Solomon "a devshop will start on TEs on the explorer soon -- just writing up the SoW".